src/allotropy/allotrope/schemas/adm/solution-analyzer/REC/2024/03/CHANGE_NOTES.md

```diff
@@ -0,0 +1,3 @@
+Base schema: http://purl.allotrope.org/json-schemas/adm/solution-analyzer/REC/2024/03/solution-analyzer.schema
+
+This is a proposed new schema for solution-analyzer devices, including an extension to the ASM model for the Data System Document.
```

Comment:
You should remove the language about proposed new schema, since we have a base schema here.

Please pattern match this off of an existing CHANGE_NOTES for format, e.g. https://github.com/Benchling-Open-Source/allotropy/blob/main/src/allotropy/allotrope/schemas/adm/cell-counting/BENCHLING/2023/11/CHANGE_NOTES.md